### ISSUE TITLE
Pasting from unloaded file crashes OMEdit

### ISSUE THREAD

**Issue by ceraolo** [OpenModelica/OpenModelica#11516]
### Description
Pasting a shape from an unloaded file crashes OMEdit

### Steps to Reproduce
1) load T1: 
```
model T1
equation

annotation(
    Diagram(graphics = {Ellipse(origin = {-2, 33}, extent = {{-42, 31}, {42, -31}})}));
end T1;
```
2) copy the ellipse
3) unload T1
3) load T2

```
model T2
equation

annotation(
    Diagram(graphics = {Rectangle(origin = {-6, -38}, extent = {{-38, 18}, {38, -18}})}));
end T2;
```
5) paste => CRASH

### Expected Behavior
should possibly paste. And if it cannot, send an error message (avoid any crash).

### Version and OS
OM 1.22.0 beta6 (with encryption)

 - OpenModelica Version: [e.g. `omc --version` or `Help->About OMEdit` from OMEdit]
 - OS: [e.g. Windows 10, 64 bit]
 - Versions of used Modelica libraries if applicable

**Comment by adeas31** [comment]
I updated so it will not crash now. The model will be removed from the clipboard when its unloaded.

**Comment by ceraolo** [comment]
This solves the main issue, i.e. che program crash.

It would be nice if, at least in simple cases, clipboard is kept instead (Dymola does this). 
If this is done only in some cases, whenever one wants to paste some clipboard content not yet available, a message could be issued saying this.

Use case: since we cannot load simultaneously packages having the same name, if we happen to have two different versions of the same package (it happens, from time to time), we might want to transfer pieces of one version into the other. Copying/pasting can do the job, but before loading the new file we must unload the old one, hoping that the clipboard content is kept.

**Comment by adeas31** [comment]
I think it is possible to keep the clipboard. We might need to add some support for it in the interactive environment. Please make  a new ticket about it.

> Use case: since we cannot load simultaneously packages having the same name, if we happen to have two different versions of the same package (it happens, from time to time), we might want to transfer pieces of one version into the other. Copying/pasting can do the job, but before loading the new file we must unload the old one, hoping that the clipboard content is kept.

I don't really understand what you mean.

**Comment by ceraolo** [comment]
>I think it is possible to keep the clipboard. We might need to add some support for it in the interactive environment. Please make a new ticket about it.

Good to know. I'll do.

> I don't really understand what you mean.

It may happen to have the same package Pkg on two computers (PC1 and PC2). It has happened to me that I made different changes on the two files. So, I would decide to copy the changes made on Pkg in PC1 and paste them into Pkg changed in PC2. To do this I need keeping clipboard content when I dowloaded the first parckage (after copying), before pasting into the new one.

Is this frequent? I don't know, but just in case it is not too complicated to implement, keeping the clipboard across sessions is useful here.
